### PR TITLE
fix add

### PR DESCRIPTION
**Pull request by pineman**

### CODE REVIEW

File: tests/lr35902.rs [hunk 1]
```diff
@@ -82,9 +84,7 @@ impl LR35902Builder {
 fn test_instr_0x000_nop() {
     let builder = LR35902Builder::new().with_memory_byte(0, 0);
     let mut sut = builder.clone().build();
-
```
Comment (joajfreitas) [inline tests/lr35902.rs:85]: I really much prefer to keep the separation between setup, effects, expectations

File: src/lr35902.rs [hunk 2]
```diff
@@ -151,19 +141,19 @@ impl LR35902 {
     }
 
     fn set_z_flag(&mut self, value: bool) {
-        self.af = bw::set_bit16::<8>(self.af, value);
+        self.af = bw::set_bit16::<7>(self.af, value);
```
Comment (joajfreitas) [inline src/lr35902.rs:141]: uff

**Review by joajfreitas — CHANGES_REQUESTED** [review]

File: src/lr35902.rs (hunk 2 above)
Comment (pineman) [reply src/lr35902.rs:141]: insert off by one joke

File: tests/lr35902.rs (hunk 1 above)
Comment (pineman) [reply tests/lr35902.rs:85]: it was in part to test the stack overflow bug

**Comment by pineman** [comment]
@joajfreitas lets merge this?

**Pull request merged**